### ISSUE TITLE
User cannot input character `c`, `m`, and `q` to Client ID and secret field

### ISSUE THREAD

**Issue by rcl1** [ck-zhang/reddix#2]
### 1. Description
When adding an account, user cannot input characters `c`, `m`, and `q` to Client ID and Secret field because it clashes with keyboard navigations.

### 2. Steps to Reproduce
1. Open Reddix
2. Open the guided menu with `m`
3. Go to `Add new account...`
4. Users with their client ID or secret with character `c`, `q` and `m` both in lowercase and uppercase cannot insert the letter because Reddix interprets it as `c copy link`, `m guided menu` or `q quit`.

### 3. Expected Behavior
Reddix should understand the input as the character `c`, `q` and `m` not the corresponding keyboard navigations.

### 4. Screenshots
<img width="1117" height="270" alt="Image" src="https://github.com/user-attachments/assets/94ee5bc4-33d6-41df-8971-6e4d05760438" />

### 5. Environment
- OS: Windows 10 Pro 22H2
- Terminal: Powershell
- Version: 0.1.5

**Comment by HeyUKonrad** [comment]
having m in the secret leads to the same issue btw

**Comment by ck-zhang** [comment]
Fixed that in Commit c996ce0, try the latest version to see if that's fixed. I don't have a windows machine nearby to test it out.

**Comment by githubaccount073** [comment]
> Fixed that in v0.1.8, try the latest version to see if that's fixed. I don't have a windows machine nearby to test it out.

q still wont work in the name or secret field.
